blackbox/docs/release_notes/2.0.0.txt

```diff
@@ -299,3 +302,19 @@ sets both the minimum and maximum memory to allocate, and should be set to
 whatever your previous ``CRATE_MAX_MEM`` was set to.
 
 .. _enterprise edition: https://crate.io/enterprise-edition/
+
+Cluster name in path data
+-------------------------
+
+The computation of the effective data directory path has changed in a way that
```

Comment:
I just talked to @joemoe about that. Since this currently is 'just' deprecated, there is no need to add it to the changes, or current release notes:
1. We added a cluster check, that informs the user and is backported to 2.1
2. When we upgrade to ES >= 6.0 which will cause the fallback to be removed, another note about this should be added to the release notes